src/awkward/_nplikes/typetracer.py

```diff
@@ -678,7 +557,9 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
         #         "TypeTracerArray objects should not be used directly with ufuncs"
         #     )
         # )
-        if method != "__call__" or len(inputs) == 0 or "out" in kwargs:
+        kwargs.pop("out", None)
+
+        if method != "__call__" or len(inputs) == 0:
```

Comment:
I think this is harmless, and we need it for in-place operations upon scalars (though we could also disable that in the mixin)